src/web/mjs/connectors/ScanManga.mjs

```diff
@@ -72,6 +72,32 @@ export default class ScanManga extends Connector {
         }));
     }
 
+    async _getPages(chapter) {
+        let uri = new URL(chapter.id, this.url);
+        uri.searchParams.set('style', 'list');
+        let request = new Request(uri, this.requestOptions);
+        let data = await this.fetchDOM(request, '.aLN');
+        return data.length > 0 ? this._getPagesNovel(request) : this._getPagesManga(chapter);
+    }
+
+    async _getPagesNovel(request) {
+        let script = `
+            new Promise(resolve => {
+                document.body.style.width = '56em';
+                let novel = document.querySelector('article.aLN');
+                novel.style.padding = '1.5em';
+                let script = document.createElement('script');
+                script.onload = async function() {
+                    let canvas = await html2canvas(novel);
```

Comment:
May wrap method body in `try ... catch ...` to process any unexpected error and reject the promise immediately